security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,25 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-5-0"]
+== OpenShift Compliance Operator 1.5.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.5.0:
+
+* link:https://access.redhat.com/errata/RHBA-2024:3533[RHBA-2024:3533 - OpenShift Compliance Operator 1.5.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-5-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* As of this release, the Compliance Operator provides a unique profile ID for easier programmatic use.  (link:https://issues.redhat.com/browse/CMP-2450[*CMP-2450*])
+
+* With this release, the Compliance Operator provides PCI-DSS-NODE scans and CIS-NODE profile scans for ROSA HCP. (link:https://issues.redhat.com/browse/CMP-1544[*CMP-1554*])
+
+[id="compliance-operator-1-5-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, for ROSA HCP systems, profile listings were incorrect. This update allows Compliance Operator to provide correct profile output. (link:https://issues.redhat.com/browse/OCPBUGS-34535[*OCPBUGS-34535*])
```

Comment:
We could supplement this by saying something to the effect of:

"The Compliance Operator will only load `Node` profiles when running on ROSA HCP, since it is a Red Hat managed platform and restricts access to the control plane, making `Platform` profiles out-of-scope."

Reply:
@rhmdnd I've added the text with slight worthsmithy changes...